src/main/java/org/app/controller/TodoController.java

```diff
@@ -29,9 +33,31 @@ public ResponseEntity<TaskResponseDto> getTask(@PathVariable("id") Long id, Auth
         return ResponseEntity.ok(task);
     }
 
+    @GetMapping("/all")
+    public ResponseEntity<List<TaskResponseDto>> getAllTasks(Authentication authentication) {
+        List<TaskResponseDto> tasks = todoServiceImpl.getAllTasks(authentication.getName());
+        return ResponseEntity.ok(tasks);
+    }
+
+    @GetMapping("/filter")
```

Comment:
Calling `GET http://localhost:8080/api/user/todos/filter?completed=true` returns all tasks (completed and uncompleted)